### ISSUE TITLE
Path must be a string. Received undefined

### ISSUE THREAD

**Issue by nightcoder11** [cucumber/cucumber-js#1159]
I'm having issues running coverage report after upgrading Cucumber to any versions from 3.0.0.

Below is the stack trace and line 30 of commonStepDefinitions has the function Given('I have a valid API Key', function () {
	this.requestOptions.headers['x-api-key'] = this.apiKey;
});

-----------------------
npm run coverage

> enotify@1.0.0 coverage C:\ws\myproject
> istanbul cover node_modules/mocha/bin/_mocha -- -R spec --recursive

No coverage information was collected, exit without writing coverage information
path.js:7
    throw new TypeError('Path must be a string. Received ' + inspect(path));
    ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.relative (path.js:538:5)
    at getDefinitionLineAndUri (C:\ws\myproject\node_modules\cucumber\lib\support_code_library_builder\define_helpers.js:146:28)
    at C:\ws\myproject\node_modules\cucumber\lib\support_code_library_builder\define_helpers.js:110:34
    at Object.<anonymous> (C:\ws\myproject\test\integration-tests\features\common_stepdefinitions\commonStepDefinitions.js:30:1)

**Comment by charlierudolph** [comment]
What version of cucumber are you on? I believe this issue was fixed already in a newer version

**Comment by znevrly** [comment]
Also check this https://github.com/cucumber/cucumber-js/issues/1140#issuecomment-426872896

**Comment by aslakhellesoy** [comment]
I'm also seeing this. I'm on 5.0.3.

**Comment by aslakhellesoy** [comment]
I had `npm link`'ed another repo that also depends on `cucumber`. Unlinking it made the error go away.

**Comment by slvganesh** [comment]
I had the same issue and mine was resolved by changing the case (from c: to C: ) in my NODE_PATH variable.

Here is the step I followed:

1. Open cmd and navigate to the folder from where you are executing your cucumber scripts
2. Run "cucumber-js" command in the prompt
3. It should produce results like "0 scenarios 0 steps" but for my case, it was like the Executed path is different from Local path
4. The path in both the Executed path and Local path must be same. If not, consider updating your NODE_PATH  (Note: Here the paths must be case sensitive)
5. Perform step 2 again and if you get "0 scenarios 0 steps" then your issue is likely to be fixed.
